spring-cloud-contract-stub-runner/src/main/java/org/springframework/cloud/contract/stubrunner/FileStubDownloader.java

```diff
@@ -20,12 +20,12 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
-import java.nio.file.Paths;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.regex.Pattern;
 
+import org.apache.commons.io.FilenameUtils;
```

Comment:
Do we need to add a new dependency just for this? I'd prefer not to introduce it.

Reply:
@marcingrzejszczak The issues I had was that on windows the org.eclipse.jgit.util.FileUtils; will not recursivly delete one of the files in .git and this this then leaves the tmp folder in tact. This fails the tests on Windows but passes on Unix.

Reply:
So I'd prefer not to add a new dependency but write our own version of the deleting mechanism. There is the FileSystemUtils class in spring that can be helpful

Reply:
@marcingrzejszczak This is now removed. Using the Spring version that you suggested with works